### DISCUSSION TITLE
Update bootloader problem with key nolapic

### DISCUSSION THREAD

**Discussion by Sergmasharov** [home-assistant/operating-system#2667] [General]
### Describe the issue you are experiencing

I have a special device. It boot any linux only with -nolapic key.
I have to edit EFI/Boot/grub.cfg file and to bootlines nolapic key and it works. After updating HA OS it updates this file and my system not able to boot. 
So after every update i neet to extract my disk then insert it to another pc, mount volume and edit grub.cfg It is boooring....
How I can fix nolapic parameter in bootloader?

### What operating system image do you use?

generic-x86-64 (Generic UEFI capable x86-64 systems)

### What version of Home Assistant Operating System is installed?

10.4

### Did you upgrade the Operating System.

Yes

### Steps to reproduce the issue

1.
2.
3.
...

### Anything in the Supervisor logs that might be useful for us?

```txt
no errors
```

### Anything in the Host logs that might be useful for us?

```txt
no errors
```

### System information

[core-ssh ~]$ cat /proc/cpuinfo
processor : 0
vendor_id : GenuineIntel
cpu family : 6
model : 54
model name : Genuine Intel(R) CPU @ 1.86GHz
stepping : 1
microcode : 0x10d
cpu MHz : 1861.949
cache size : 512 KB
physical id : 0
siblings : 1
core id : 0
cpu cores : 1
apicid : 0
initial apicid : 0
fpu : yes
fpu_exception : yes
cpuid level : 10
wp : yes
flags : fpu vme de pse tsc msr pae mce cx8 sep mtrr pge mca cmov pat pse36 clflush dts acpi mmx fxsr sse sse2 ss ht tm pbe syscall nx lm constant_tsc arch_perfmon pebs bts nopl nonstop_tsc cpuid aperfmperf pni dtes64 monitor ds_cpl est tm2 ssse3 cx16 xtpr pdcm movbe lahf_lm dtherm arat
bugs :
bogomips : 3723.77
clflush size : 64
cache_alignment : 64
address sizes : 36 bits physical, 48 bits virtual
power management:

**Comment by agners** [comment]
You can edit the `cmdline.txt` file (located in `/mnt/boot/`) and append `nolapic` there. The update script do not overwrite that file.

**Reply by Sergmasharov** [reply]
Thank you for advice!
